index.html

```diff
@@ -67,8 +73,8 @@ <h2 class="salutation">Tonic</h2>
 <!-- The end if card 1 -->
 
 <!-- card 2 -->
-<div class="card">
-    <div class="flex-item"><img src="images/Snapshoot Portfolio.png" alt="works 1"></div>
+<div class="card card-reverse">
+    <div class="flex-item"><img class="card-img2" src="images/Snapshoot Portfolio.png" alt="works 1"></div>
```

Comment:
- [ ] As requested by the previous reviewer please make sure that buttons has less padding and it matches the given design and also make sure that you adjust the spacing.
- [ ] Also increase the font size of the button to match the Fimga design previously it was good.
- [ ] Please implement to all the section.
- [ ] [Optional] I would suggest please make your list similar to the Figma design.
### Your design
![his](https://github.com/ausiemass/portfolio-austin/assets/81727988/15590fef-178d-4070-9c70-816aa59f4570)

### Figma design
![fimga](https://github.com/ausiemass/portfolio-austin/assets/81727988/e0e04048-1b28-4b77-b117-7e30da4ffc7b)